main/options/options.cc

```diff
@@ -320,6 +320,11 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
         "End users should prefer to use `--enable-all-beta-lsp-features`, instead.)");
     options.add_options("advanced")("enable-all-beta-lsp-features",
                                     "Enable (expected-to-be-non-crashy) early-access LSP features.");
+    options.add_options("advanced")(
+        "lsp-max-errors-reported",
+        "Limits the maximum number of simultaneous errors that LSP reports to the editor. Can prevent "
+        "editor slowdowns with large error lists.",
```

Comment:
You don't mention this here, but you do in the comment: "0 means no limit". is that the behavior? (it looks like it is from the code, but I don't see any tests)

Reply:
Yes; i should mention this here -- thanks!

(How do you test the lack of a limit? I guess I can see about generating a large number of errors.)

Reply:
The other option for zero would have been "reports no errors" so if you say 0 and it reports any errors, that's fine by me.

Reply:
ah, smart!